### PR TITLE
Fix logger-plugin module directory structure problem(Maven cannot trigger junit testing).

### PR DESCRIPTION
**Pull request by zifeihan**
### Fix Fix logger-plugin module directory structure problem(Maven cannot trigger junit testing).
- [ ] Add a unit test to verify that the fix works.
- [x] Explain briefly about why the bug exists and how to fix it.

@vcjmhg 
Fix logger-plugin module directory structure problem(Maven cannot trigger junit testing), and format the test code.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Release Notes.
 * The operation name of quartz-scheduler plugin, has been changed as the `quartz-scheduler/${className}` format.
 * Fix jdk-http and okhttp-3.x plugin did not overwrite the old trace header.
 * Support collecting logs of log4j, log4j2, and logback in the tracing context with a new `logger-plugin`.
+* Fix logger-plugin module directory structure problem(Maven cannot trigger junit testing).
```
Comment (wu-sheng) [inline CHANGES.md:15]:
```suggestion
```

This fix is not related to the changelog, as the bug codes were introduced in the master only. Please remove.

**Review by wu-sheng — DISMISSED** [review]

**Review by wu-sheng — APPROVED** [review]

**Review by wu-sheng — APPROVED** [review]

File: apm-sniffer/optional-plugins/logger-plugin/src/test/java/org/apache/skywalking/apm/plugin/logger/ContextConfigDefaultTest.java [hunk 2]
```diff
@@ -24,7 +25,7 @@
 public class ContextConfigDefaultTest {
     @Test
     public void testDefaultConfig() {
-        ContextConfig config = ContextConfig.getInstance();
+        ContextConfig config = ContextConfig.getLatestConfig();
```
Comment (vcjmhg) [inline apm-sniffer/optional-plugins/logger-plugin/src/test/java/org/apache/skywalking/apm/plugin/logger/ContextConfigDefaultTest.java:28]: @zifeihan Why we should write another method named `getLastsConfig()` for UT. It seems that when any UT starts, `initContextConfig()` will be executed. Just curious :smile:

Comment (zifeihan) [reply apm-sniffer/optional-plugins/logger-plugin/src/test/java/org/apache/skywalking/apm/plugin/logger/ContextConfigDefaultTest.java:28]: @vcjmhg Hello, because maven test is in a jvm instance, the loading configuration file content is in singleton mode, so if the file content is modified in the test case, it will not take effect.

Comment (zifeihan) [reply apm-sniffer/optional-plugins/logger-plugin/src/test/java/org/apache/skywalking/apm/plugin/logger/ContextConfigDefaultTest.java:28]: @vcjmhg you can test success for every UT by manual, but it will failed in maven automatic test.

File: apm-sniffer/optional-plugins/logger-plugin/src/main/java/org/apache/skywalking/apm/plugin/logger/ContextConfig.java [hunk 3]
```diff
@@ -58,6 +58,10 @@ public static ContextConfig getInstance() {
         return HolderContextConfig.INSTANCE;
     }
 
+    public static ContextConfig getLatestConfig() {
```
Comment (wu-sheng) [inline apm-sniffer/optional-plugins/logger-plugin/src/main/java/org/apache/skywalking/apm/plugin/logger/ContextConfig.java:61]:
```suggestion
    /**
     * For testing use only.
     * Avoid to use config cache. Load the configuration every time. 
     */
    static ContextConfig getLatestConfig() {
```

Should like this?

File: apm-sniffer/optional-plugins/logger-plugin/src/test/java/org/apache/skywalking/apm/plugin/logger/ContextConfigDefaultTest.java (hunk 2 above)
Comment (vcjmhg) [reply apm-sniffer/optional-plugins/logger-plugin/src/test/java/org/apache/skywalking/apm/plugin/logger/ContextConfigDefaultTest.java:28]: ok , I got it. Thank u!

**Review by wu-sheng — APPROVED** [review]
